src/d3d12/d3d12_renderer.cpp

```diff
@@ -1348,4 +1349,9 @@ namespace wr
 		}
 	}
 
+	void D3D12RenderSystem::CreateDefaultResources()
+	{
+		m_default_cubemap = m_texture_pools.at(0)->CreateCubemap("DefaultResource_Cubemap", 2, 2, 1, wr::Format::R8G8B8A8_UNORM, false);

```

Comment:
Missing extension or something?

Reply:
No, that is just a name for the resource itself, useful for debugging